src/core.js

```diff
@@ -1076,23 +1100,6 @@ define([
         }, this.opts().core.loadDelay);
     };
 
-    fn.disableUI = function () {
-        this.flipUI(false);
-    };
-
-    fn.enableUI = function () {
-        this.flipUI(true);
-    };
-
-    fn.flipUI = function (state) {
-        var $props = this.$f.find('.fd-question-properties');
-        if (state) {
-            $props.show();
-        } else {
-            $props.hide();
-        }
-    };
-        
```

Comment:
These were consolidated into a clearer API: `showQuestionProperties` and `hideQuestionProperties`